src/core/lv2/Lv2SubPluginFeatures.cpp

```diff
@@ -154,8 +243,8 @@ QString Lv2SubPluginFeatures::description(
 const PixmapLoader *Lv2SubPluginFeatures::logo(
 	const Plugin::Descriptor::SubPluginFeatures::Key &k) const
 {
-	(void)k; // TODO
-	return nullptr;
+	(void)k;
+	return nullptr; // Lv2 currently does not support this
```

Comment:
Why the changes in comment? Also the C style cast seems to have no place here. The cast looks fine to remove, or is it to suppress unused warnings?

Reply:
The change is done because I previously thought we could implement this later. So I now wanted to implement this function, but found out Lv2 does not support it. Thus, I updated the comment.